package.json

```diff
@@ -2,16 +2,65 @@
   "name": "jitsi-meet-electron",
   "version": "0.0.0",
   "description": "Electron application for Jitsi Meet",
-  "main": "build/main.js",
+  "main": "./build/main.js",
   "productName": "Jitsi Meet",
   "scripts": {
     "start": "webpack --config ./webpack.config.js --mode development && electron ./build/main.js",
-    "clean": "rm -rf node_modules",
+    "clean": "rm -rf node_modules build dist",
     "lint": "eslint . && flow",
-    "pack": "webpack --config ./webpack.config.js --mode production && electron-packager .",
-    "postinstall": "electron-rebuild",
+    "build": "webpack --config ./webpack.config.js --mode production",
+    "pack": "npm run build && electron-builder --dir",
+    "dist": "npm run build && electron-builder",
+    "postinstall": "electron-builder install-app-deps",
     "validate": "npm ls"
   },
+  "build": {
+    "appId": "org.jitsi.jitsi-meet",
+    "productName": "Jitsi Meet",
+    "files": [
+      "**/*",
+      "resources",
+      "!app",
+      "!main.js"
+    ],
+    "copyright": "",
+    "mac": {
+      "category": "public.app-category.video",
+      "artifactName": "${productName}-${version}-${arch}.${ext}"
+    },
+    "linux": {
+      "artifactName": "${productName}-${version}-${arch}.${ext}",
+      "category": "VideoConference;AudioVideo;Audio;Video;Network",
+      "description": "Jitsi Meet Desktop App",
+      "executableName": "jitsi-meet",
+      "target": [
+        "AppImage",
```

Comment:
let's do only AppImage for now, since that works everywhere.

Reply:
I was thinking let's do as many as possible with minimal configuration.
Since they are already working, I don't see a reason not to include them. WDYT?

Reply:
If we publish those binaries we set the expectation that they work. However, realistically, we are not going to test them all,plus we open the door for questions like "why don't you provide RPM packages?". So, I'd say, let's start small, just AppImage, which works everywhere, and add more in the future if necessary. Adding stuff is easy, removing stuff once it's out there... not so much.

Reply:
>  we open the door for questions like "why don't you provide RPM packages?"

Yeah, That will be annoying. :sweat_smile: 

>  just AppImage, which works everywhere

If that's the reason, how about a snap package? 
I find snap packages more easy to install. (That may be because I am using Ubuntu)
Also, if I don't specify target it builds AppImage and snap package. Maybe I can remove this and let `electron-builder` use default?

Reply:
> If that's the reason, how about a snap package?
> I find snap packages more easy to install. (That may be because I am using Ubuntu)

Then the Fedora people will say: "why not Flatpak?" It's the same thing all over again :-)

> Also, if I don't specify the target, it builds AppImage and snap package. Maybe I can remove this and let electron-builder use default?

I'd rather be explicit about what we want: AppImage and nothing else.

> Other problems with AppImage
> 
>     Have to move .AppImage to organise them
>     If we move it after desktop integration. it will break.

Move it, how? You mean move the file location? If that's the case, that is not a problem we create, but a problem inherent to AppImage, so I'd say we put a line in the documentation and call it a day :-)

Reply:
Cool. I have updated this.